api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAuthorsToInformationResourceGenerator.java

```diff
@@ -109,6 +109,14 @@ public String getN3PrefixString() {
 		 "@prefix foaf: <" + foaf + "> .  \n"   ;
 	}
 
+    public String getDisableVCardPrefix() {
+        return "@prefix fail_pattern: ?createVCard .\n";
+    }
+
+    public String getDisableRealPersonPrefix() {
```

Comment:
Could this method be named better? What does RealPerson indicate?

Reply:
Renamed method